omero/sysadmins/config.rst

```diff
@@ -1138,10 +1138,30 @@ Default: `60`
 Performance
 -----------
 
+.. property:: omero.sessions.max_user_time_to_idle
+
+omero.sessions.max_user_time_to_idle
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Sets the maximum duration a user can request before a login
+is required due to inactivity.
```

Comment:
In seconds? Milliseconds?

Reply:
That will have to be fixed in openmicroscopy